HPXMLtoOpenStudio/resources/waterheater.rb

```diff
@@ -2262,11 +2262,11 @@ def self.unmet_wh_loads_program(model, water_heating_systems, plantloop_map, sho
       unmet_wh_loads_program = OpenStudio::Model::EnergyManagementSystemProgram.new(model)
       unmet_wh_loads_program.setName(Constants.ObjectNameUnmetLoadsProgram)
       unmet_wh_loads_program.addLine("If (#{shower_flow_sensor.name} > 0) && (#{wh_temp_sensor.name} < #{mixed_setpoint_sensor.name})")
-      unmet_wh_loads_program.addLine('Set ShowerSagTime=SystemTimeStep')
-      unmet_wh_loads_program.addLine("Set ShowerE=#{shower_flow_sensor.name} * #{showers_peak_flows[water_heating_system.id]} * 4141170 * (#{mixed_setpoint_sensor.name} - #{wh_temp_sensor.name})")
+      unmet_wh_loads_program.addLine('Set ShowerSagTime = SystemTimeStep')
+      unmet_wh_loads_program.addLine("Set ShowerE = #{shower_flow_sensor.name} * #{showers_peak_flows[water_heating_system.id]} * 4141170 * (#{mixed_setpoint_sensor.name} - #{wh_temp_sensor.name})")
       unmet_wh_loads_program.addLine('Else')
-      unmet_wh_loads_program.addLine('Set ShowerSagTime=0')
-      unmet_wh_loads_program.addLine('Set ShowerE=0')
+      unmet_wh_loads_program.addLine('Set ShowerSagTime = 0.0')
+      unmet_wh_loads_program.addLine('Set ShowerE = 0.0')
```

Comment:
Can we revert this change and fix the root cause? We should be able to handle the lack of spaces. It'd be nice to not have a developer hit this issue again in the future.